examples/supertuxkart/Dockerfile

```diff
@@ -26,35 +24,22 @@ RUN go build -o wrapper .
 # compile stk
 FROM debian:buster as stk-builder
 
-RUN apt update && apt-get install -y build-essential cmake libbluetooth-dev \
-libcurl4-openssl-dev libenet-dev libfreetype6-dev libharfbuzz-dev libfribidi-dev \
-libgl1-mesa-dev libglew-dev libjpeg-dev libogg-dev libopenal-dev libpng-dev \
-libssl-dev libvorbis-dev libxrandr-dev libx11-dev nettle-dev pkg-config zlib1g-dev \
-git subversion
+RUN apt update && apt-get install -y  wget xz-utils
 
-RUN git clone -b allow-ai https://github.com/markmandel/stk-code.git
-RUN svn co https://svn.code.sf.net/p/supertuxkart/code/stk-assets stk-assets
-RUN rm -rf ./stk-code/.git
-RUN rm -rf ./stk-assets/.svn
+RUN wget https://github.com/supertuxkart/stk-code/releases/download/1.3/SuperTuxKart-1.3-linux-64bit.tar.xz
 
-RUN cd stk-code && mkdir cmake_build && cd cmake_build \
-    && cmake .. -DBUILD_RECORDER=off -DSERVER_ONLY=ON \
-    && make -j$(nproc)
+RUN tar -xf SuperTuxKart-1.3-linux-64bit.tar.xz
+RUN rm -rf SuperTuxKart-1.3-linux-64bit.tar.xz
+RUN ./SuperTuxKart-1.3-linux-64bit/run_game.sh
```

Comment:
I just tried to build this image, and noticed that you seem to be running the game here (which fails). I'm not 100% sure why?